modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GPAD_conf.pm

```diff
@@ -139,43 +138,114 @@ sub pipeline_analyses {
     {
       -logic_name        => 'AnalysisSetup',
       -module            => 'Bio::EnsEMBL::Production::Pipeline::Common::AnalysisSetup',
       -max_retry_count   => 0,
+      -analysis_capacity => 20,
       -parameters        => {
                               db_backup_required => 0,
+                              delete_existing => $self->o('delete_existing_analysis'),
                               production_lookup  => $self->o('production_lookup'),
                               logic_name => $self->o('logic_name'),
                               description => $self->o('description'),
                               display_label => $self->o('display_label'),
                               displayable => $self->o('displayable'),
-                              delete_existing => $self->o('delete_existing'),
                               db => $self->o('db'),
                               program => $self->o('program'),
+                              linked_tables => $self->o('linked_tables')
+                            }
+    },
+    {
+      -logic_name        => 'RemoveOrphans',
+      -module            => 'Bio::EnsEMBL::Production::Pipeline::Common::SqlCmd',
+      -max_retry_count   => 0,
+      -analysis_capacity => 20,
+      -parameters        => {
+                              sql => [
+                                'DELETE onx.* FROM '.
+                                  'ontology_xref onx LEFT OUTER JOIN '.
+                                  'object_xref ox USING (object_xref_id) '.
+                                  'WHERE ox.object_xref_id IS NULL',
+                              ]
                             },
+    },
+        {
+      -logic_name        => 'SpeciesFactory',
+      -module            => 'Bio::EnsEMBL::Production::Pipeline::Common::DbAwareSpeciesFactory',
+      -max_retry_count   => 1,
+      -parameters        => {
+                              chromosome_flow    => 0,
+                              otherfeatures_flow => 0,
+                              regulation_flow    => 0,
+                              variation_flow     => 0,
+                            },
+      -flow_into         => {
+                              '2' => ['FindFile'],
+                            }
+    },
+    { -logic_name     => 'FindFile',
+      -module         => 'Bio::EnsEMBL::Production::Pipeline::GPAD::FindFile',
+      -analysis_capacity  => 30,
+      -rc_name 	      => 'default',
+      -parameters     => {
+                            gpad_directory => $self->o('gpad_directory')
+      },
+      -flow_into         => {
+                  '2' => ['gpad_file_load'],
+                }
     },
     { -logic_name     => 'gpad_file_load',
   	  -module         => 'Bio::EnsEMBL::Production::Pipeline::GPAD::LoadFile',
-      -hive_capacity  => 10,
-      -rc_name 	      => 'default',     
-    },   
-
+      -parameters     => {
+                              delete_existing => $self->o('delete_existing_go'),
```

Comment:
Might be worth passing in the logic_name here, rather than having 'goa_import' hard-coded in the module?

Reply:
That's a good point, will do that.

Reply:
see latest commit